infrastructure/main.tf

```diff
@@ -84,6 +84,11 @@ data "azurerm_key_vault_secret" "robotics_email_scottish_to" {
   vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
 }
 
+data "azurerm_key_vault_secret" "robotics_email_pip_ae_to" {
+  name      = "robotics-email-pip-ae-to"
+  vault_uri = "${data.azurerm_key_vault.sscs_key_vault.vault_uri}"
```

Comment:
have you put this value in the Vault??

Reply:
No not yet put, I might need your help on this

Reply:
The key Vault is created on AAT and PROD